### ISSUE TITLE
Inability to puff up one´s cheeks

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10568]
**Preferred term label:**
Inability to puff up one´s cheeks

**Synonyms**
none

**Definition (free text, please give PubMed ID)**
Due to facial muscle weakness inability to puff up one´s cheeks. (https://doi.org/10.1007/s00415-020-10281-z)

**Parent term (use hpo.jax.org/app)**
Weakness of facial musculature HP:0030319

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:269

**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling 
Not an expert but I think that inability to puff cheeks can not only be due to facial muscle weakness, but also to Cranial Nerve VII lesions -- https://www.ncbi.nlm.nih.gov/books/NBK585066/

**Comment by pnrobinson** [comment]
@CorneliusDreyling 

Adding new term: 
- ID: HP:6001102
- label: Inability to puff cheeks
- parents: [Weakness of facial musculature (HP:0030319)](https://hpo.jax.org/app/browse/term/HP:0030319)
- pmid: 
- def: Reduced ability to inflate the cheeks by blowing air into them while keeping the lips closed.
- comment: This feature may be related to facial muscle weakness as well as to cranial nerve VII lesions.
- orcid: orcid:0009-0004-9942-3886
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10568
- synonyms:
